From 5a4d97181f66cafb0ab9780d841f36e8879b5ed0 Mon Sep 17 00:00:00 2001
From: Alisher Alikhodjaev <alisher@google.com>
Date: Thu, 17 Jun 2021 11:27:00 -0700
Subject: [PATCH] Add HIDE_NON_SYSTEM_OVERLAY_WINDOWS permission to Nfc

This permission is required for xx_FLAG_HIDE_NON_SYSTEM_OVERLAY_WINDOWS
window flag to function properly.

Bug: 180422108
Test: build ok
Change-Id: I8246c06c0e7d60d4e06ef4fa430ccd5111e99a43
(cherry picked from commit 9c56b01c5745252c13c05a2fe39faaef130813e5)
---
 AndroidManifest.xml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index f6a07166..fba2509d 100755
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -35,6 +35,7 @@
     <uses-permission android:name="android.permission.READ_CONTACTS" />
     <uses-permission android:name="android.permission.READ_FRAME_BUFFER" />
     <uses-permission android:name="android.permission.SYSTEM_ALERT_WINDOW" />
+    <uses-permission android:name="android.permission.HIDE_NON_SYSTEM_OVERLAY_WINDOWS" />
     <uses-permission android:name="android.permission.ACCESS_WIFI_STATE" />
     <uses-permission android:name="android.permission.CHANGE_WIFI_STATE" />
     <uses-permission android:name="android.permission.VIBRATE" />
-- 
2.17.1

